test/unit/mocks/store.js

```diff
@@ -1,20 +1,16 @@
-const sinon = require('sinon')
+const mockElectron = require('./electron')
+const mockLogger = require('./logger')
+const Store = require('electron-store')
+const proxyquire = require('proxyquire').noCallThru()
 
 module.exports = function mockStore () {
-  let store = {}
-
-  return {
-    get: sinon.stub().callsFake((key, def) => {
-      return typeof store[key] !== 'undefined' ? store[key] : def
-    }),
-    set: sinon.stub().callsFake((key, val) => {
-      store[key] = val
-    }),
-    clear: () => {
-      store = {}
-    },
-    get store () {
-      return store
-    }
+  function MockElectronStoreConstructor ({ ...options }) {
+    return new Store({ ...options, migrations: undefined })
```

Comment:
using the real electron-store, but removing any migrations to prevent CI failures.